service/gcs/core/gcs/storage.go

```diff
@@ -185,14 +190,14 @@ func scsiControllerLunToName(osl oslayer.OS, controller, lun uint8) (string, err
 
 // scsiLunToName finds the SCSI device with the given LUN. This assumes
 // only one SCSI controller.
-func scsiLunToName(osl oslayer.OS, lun uint8) (string, error) {
-	return scsiControllerLunToName(osl, 0, lun)
+func (c *gcsCore) scsiLunToName(lun uint8) (string, error) {
```

Comment:
This was V1 only. Marking that explicitly here. If we want this a separate change I can do that.